R/class-workbook-utils.R

```diff
@@ -158,6 +158,15 @@ next_sheet <- function() {
   structure("next_sheet", class = "openxlsx2_waiver")
 }
 
+na_strings <- function() {
```

Comment:
Since this is visible to the user, it probably should have a corresponding man entry?

Reply:
It might make sense to export the other waivers, too.  Currently `current_sheet()` and `next_sheet()` are also internals only.

Reply:
Yes, good idea. Unfortunately documentation is not our strongest side :)